clustering/__init__.py

```diff
@@ -4,53 +4,25 @@
 from sklearn.decomposition import PCA
 from sklearn.cluster import KMeans
 
-
-def find_incident(labels):
-    res = set()
-    for i in range(len(labels)):
-        if labels[i].startswith('i_'):
-            res.add(i)
-    return res
-
-
-# numpy array construction
-def construct_np_allin(allin, m, n, incident_labels):
-    np_allin = []
-    for i in range(m):
-        tmp = []
-        count = 0
-        for j in range(n):
-            if allin[i][j] == True:
-                tmp.append(1)
-            elif allin[i][j] == False:
-                tmp.append(0)
-            elif allin[i][j] != 'None':
-                tmp.append(1)
-            else:
-                tmp.append(0)
-            if j in incident_labels and allin[i][j] == 1:
-                count += 1
```

Comment:
Why are we counting the incident values? And why only if the input value is `1`? It doesn't make sense to have it restricted just to `1`s on the original input matrix. I'm not sure why to sum the values either? How does the clustering benefit from that since all the columns summed are already present? Was this verified to be needed?

Reply:
Lol, I've just noticed, that the count is not used at all, since it's later removed on line [47](https://github.com/RedHatInsights/aicoe-insights-clustering/pull/9/files#diff-2a440676ce12f1a1b39194070ef96a56L47) via `np_allin[:, :-1]`. In that case I don't need the `counts` variable (in the new code) at all.

Reply:
🤣 yeah. As said, I just copy and pasted the code from the notebook.